content/algorithms/expected_consensus/_index.md

```diff
@@ -185,6 +401,6 @@ A single consensus fault results into:
 
 ### Detection and Reporting
 
-A node that detects and report a consensus fault is called "slasher", any user in Filecoin can be a slasher. They can report consensus faults by calling the `ReportConsensusFault` on the `StorageMinerActor` of the faulty miner. The slasher is rewarded with a portion of the offending miner's [Pledge Collateral](storage_power_actor#pledge-collateral) for notifying the network of the consensu fault.
+A node that detects and report a consensus fault is called "slasher", any user in Filecoin can be a slasher. They can report consensus faults by calling the `ReportConsensusFault` on the `StorageMinerActor` of the faulty miner. The slasher is rewarded with a portion of the offending miner's [Pledge Collateral](miner_collaterals) for notifying the network of the consensu fault.
```

Comment:
> The slasher is rewarded with a portion of the offending miner's [Pledge Collateral](miner_collaterals) for notifying the network of the consensu fault.

This is not true anymore. I think that now the slasher is rewarded with a portion of the offending miner block reward. Please check this!